vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -302,3 +302,42 @@ def test_graph_build_with_description(self, standard_px_chart):
             overlay_style={"visibility": "visible", "opacity": 0.3},
         )
         assert_component_equal(graph, expected_graph, keys_to_strip={"id"})
+
+    def test_graph_build_with_extra(self, standard_px_chart):
+        graph = vm.Graph(
+            figure=standard_px_chart,
```

Comment:
If you're able to easily get rid of the `keys_to_strip={"id"}` below, just by specifying the ID here, then let's do that. 
```suggestion
        graph = vm.Graph(
            id="graph",
            figure=standard_px_chart,
```

Reply:
Stripping id is consistent with how it's handled in other tests, so I would either update it everywhere for consistency or leave it as is.